### ISSUE TITLE
Foot mass

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10844]
**Preferred term label:**
Foot mass

**Synonyms**

**Definition (free text, please give PubMed ID)**
Lump on the foot; bump on the foot; foot growth; nodule on foot; foot tumor; foot cyst

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0001760

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Giant cell tumor of the tendon sheath (GCTTS)
Lipoma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001164
- label: Foot mass
- parents: [Abnormal foot morphology (HP:0001760)](https://hpo.jax.org/app/browse/term/HP:0001760)
- pmid: 
- def: An abnormal enlargement or swelling localized to a foot.
- comment: The word mass is usually used at an early stage of the diagnostic workup before the precise nature of the swelling has been determined.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10844
- synonyms: Bump on foot(NONE)|Lump on foot(NONE)|Foot growth(NONE)
